pkg/tenant/db/members.go

```diff
@@ -17,16 +17,24 @@ import (
 const MembersNamespace = "members"
 
 type Member struct {
-	OrgID    ulid.ULID `msgpack:"org_id"`
-	ID       ulid.ULID `msgpack:"id"`
-	Name     string    `msgpack:"name"`
-	Role     string    `msgpack:"role"`
-	Created  time.Time `msgpack:"created"`
-	Modified time.Time `msgpack:"modified"`
+	OrgID        ulid.ULID `msgpack:"org_id"`
+	ID           ulid.ULID `msgpack:"id"`
+	Email        string    `msgpack:"email"`
+	Name         string    `msgpack:"name"`
+	Role         string    `msgpack:"role"`
+	Status       string    `msgpack:"status"`
+	Created      time.Time `msgpack:"created"`
+	Modified     time.Time `msgpack:"modified"`
+	DateAdded    time.Time `msgpack:"date_added"`
+	LastActivity time.Time `msgpack:"last_activity"`
 }
 
+type MemberStatus string
+
 var _ Model = &Member{}
 
+const MemberConfirmed = "Confirmed"
```

Comment:
So I guess the states will be `Pending` and `Confirmed`? Are there any other statuses I'm missing?

Reply:
Yes, those are the only 2 statuses. Although, it is possible that we could add more, like `Inactive` for example.